core/common/src/geometry/Placement.ts

```diff
@@ -40,6 +40,8 @@ export type ElementAlignedBox2d = Range2d;
  */
 export type LocalAlignedBox3d = Range3d;
 
+export type Placement = Placement2d | Placement3d;

```

Comment:
Add to `GeometricElementProps` a `placement: Placement` property?

Reply:
yeah, that's probably better

Reply:
done